conf/livy-env.sh.template

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to Cloudera, Inc. under one
```

Comment:
I'm not really sure the license header is needed in these empty conf files. It doesn't hurt to leave them, though.